core/solver/multigrid.cpp

```diff
@@ -467,6 +468,118 @@ void MultigridState::run_cycle(multigrid::cycle cycle, size_type level,
 }  // namespace multigrid
 
 
+std::function<size_type(const size_type, const LinOp*)> get_selector(
+    std::string key)
+{
+    static std::map<std::string,
+                    std::function<size_type(const size_type, const LinOp*)>>
+        selector_map{
+            {{"first_for_top", [](const size_type level, const LinOp*) {
```

Comment:
I can provide another default option without settings, which I think should be more practical for most cases.
use `mg_level.at(level)` when `level < vector_len` and reuse the last one if `level >= vector_len`.
It may be more suitable in the default selector not in config

Reply:
That doesn't help. The selector should not be part of this release. It's still too complex with very little use cases.

Reply:
I have created #1611 and remove the selector now